src/docusign-po-failure/logic/logic.ergo

```diff
@@ -120,7 +119,7 @@ contract PurchaseOrderFailureContract over PurchaseOrderFailure state PurchaseOr
       promisee: some(contract.buyer),
       deadline: none,
       amount: penalty,
-      description: contract.buyer.partyId ++ " should be paid a penalty"
+      description: toString(contract.buyer) ++ " should be paid a penalty"
```

Comment:
What is the value of `toString(reference)`? Something like `resource:namespace.Type#identifier`?

Reply:
yeah. Relationships are still a pretty murky area from and Ergo standpoint, needs re-review.